Doc/library/string.rst

```diff
@@ -546,6 +546,11 @@ The available presentation types for :class:`float` and
    |         | :class:`float`, and shows all coefficient digits         |
    |         | for :class:`~decimal.Decimal`.  If ``p=0``, the decimal  |
    |         | point is omitted unless the ``#`` option is used.        |
+   |         |                                                          |
+   |         | The number of zeros padding to the exponent is           |
+   |         | determined by the implementation. When the coefficient   |
+   |         | is zero, the exponent value is also determined by the    |
+   |         | implementation.                                          |
```

Comment:
I don't think this documents something and I'm not sure if we should do something here.

For floats, we have from the C standard: "The exponent always contains at least two digits, and only as many more digits as necessary to represent the exponent.  If the value is zero, the exponent is zero."  Decimal's have no constraints on minimum number of digits in the exponent representation (in fact, to-scientific-string forbids leading zeros).  When the number is zero, the exponent value is determined here by the sign and precision.